packages/atomic/src/components/insight/atomic-insight-result-list/atomic-insight-result-list.tsx

```diff
@@ -156,6 +156,12 @@ export class AtomicInsightResultList {
     if (this.resultListState.hasError) {
       return null;
     }
+    if (
```

Comment:
Could also be moved/extracted to the piece of code that controls the placeholder display couple lines below.

ie: 
```
{!this.bindings.store.isAppLoaded() && (
  <ListDisplayResultsPlaceholder
    resultsPerPageState={this.resultPerPageState}
    display="list"
    density={this.density}
    imageSize={this.imageSize}
  />
)}
```

could become: 
```
{this.shouldDisplayPlaceholder() && (
  <ListDisplayResultsPlaceholder
    resultsPerPageState={this.resultPerPageState}
    display="list"
    density={this.density}
    imageSize={this.imageSize}
  />
)}
```

Reply:
I'd rather not render anything (returning null/undefined) than the list-wrapper, list-root, etc.

In that case you want the result list to be completely hidden.